stripe/src/main/java/com/stripe/android/view/CountryTextInputLayout.kt

```diff
@@ -125,6 +125,12 @@ internal class CountryTextInputLayout @JvmOverloads constructor(
         }
     }
 
+    override fun setEnabled(enabled: Boolean) {
+        super.setEnabled(enabled)
+        // setEnabled is called in super constructor, the empty check is required
+        countryAutocomplete?.isEnabled = enabled
```

Comment:
Is there anyway to avoid this?

Reply:
Hmm I don't like it either, since `countryAutocomplete` can only be initialize after super constructor call and `setEnabled` is called in super constructor, the `val` non-null contract breaks here.
It's possible to define it as a `var countryAutocomplete: AutoCompleteTextView?`, but that would make all of its accessors have to use `.?`, which also isn't ideal.

Finally came up with the solution to create a separate `setEnabledWithChildView` function, `CardFormView` now calls this one instead of the default one so that we don't break super constructor. I guess this is not ideal either but less ugly ☹️